iam-persistence/src/main/resources/db/migration/h2/V106__add_service_account.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE iam_account ADD COLUMN service_account BOOLEAN DEFAULT false;
+
+UPDATE iam_account SET service_account = false;
```

Comment:
Is the update necessary? I think the first statement is sufficient

Reply:
yes, second line removed